.github/workflows/ci.yml

```diff
@@ -39,12 +39,14 @@ jobs:
           cache-dependency-path: |
             **/requirements*.txt
 
-      - name: Install Dependencies
-        id: deps
+      - name: Disable man page auto-update
```

Comment:
Can you add the APT package caching here too please?

Reply:
It is already present in the code.